annex/examples/annexA/annexA.md

```diff
@@ -0,0 +1,19 @@
+ANNEX
```

Comment:
This file can be moved to `annex/annexA.md`. That way the annex folder has all the .md files, and then the examples folder has all the examples.

Reply:
Done